src/redux/reducers/backendStatus/experimentsDelete.js

```diff
@@ -0,0 +1,13 @@
+import produce from 'immer';
+// called when an experiment is deleted - deletes the backend status
+
+const backendStatusDeleted = produce((draft, action) => {
```

Comment:
Nice! But I would call this reducer the same as the action that it is reducing: `experimentsDeleted` and also. rename the file to `experimentsDeleted`